webapp/publisher/snaps/views.py

```diff
@@ -115,6 +116,11 @@ def _handle_error_list(errors):
     view_func=post_trigger_build,
     methods=["POST"],
 )
+publisher_snaps.add_url_rule(
```

Comment:
I would have a static URL, and extract the github owner and repo from the PR payload?
Having it this way means I would be able to trigger builds for someone else's snaps by setting up the URL with their repo details?

Reply:
Oh, I was generating the secret from the URL variables, so a user with his secret could trigger other people Snap with a modified payload and the secret will pass the check.

Yes, it can be static. I was trying to have the same URL that we have in build.s.i but it doesn't matter.